app/components/team-card.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+	MilitaryTech,
+	TrendingUp,
+	TrendingUpOutlined
+} from "@mui/icons-material";
+import { Box, ButtonBase, Grid, Typography } from "@mui/material";
+import React from "react";
+import MaetIcon from "./maet-icon";
+
+export type TeamCardProps = {
+	name: string;
+	location: string;
+	position: number;
+	medalColor?: string;
+	score: number;
+	change: ScoreChange;
+};
+
+export type ScoreChange = {
```

Comment:
take this out and import it, it is defined twice or more. In this file and PlayerCard comp has the exact type